doc/userdoc/model_details/HillTononiModels.ipynb

```diff
@@ -2025,6 +2025,18 @@
    "metadata": {},
    "outputs": [],
    "source": []
+  },
+ {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "License\n",
+    "-------\n",
+    "\n",
+    "This notebook (and associated files) is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 2 of the License, or (at your option) any later version.\n",
+    "\n",
+    "This notebook (and associated files) is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details."
```

Comment:
I would suggest to stick with our standard notice. In any case we need to add the copyright line. Maybe also horizontal line above the "License" to set it off from the text and a smaller heading?
```suggestion
    "-----------------------------\n",
    "### License\n",
    "\n",
    "This file is part of NEST. Copyright (C) 2004 The NEST Initiative\n",
    "\n",
    "NEST is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 2 of the License, or (at your option) any later version.\n",
    "\n",
    "NEST is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details."
```

Reply:
applied changes